src/components/menus/directory-tree-contextual-menu.tsx

```diff
@@ -385,6 +425,15 @@ const DirectoryTreeContextualMenu: React.FC<DirectoryTreeContextualMenuProps> =
                 );
             case DialogsId.ADD_NEW_CASE:
                 return <CreateCaseDialog open={true} onClose={handleCloseDialog} />;
+            case DialogsId.MOVE_DIRECTORY:
+                return (
+                    <MoveDialog
+                        open={true}
+                        onClose={handleMoveDirectory}
+                        itemsCount={1}
+                        titleId={'moveDirectoryTitle'}
```

Comment:
we must add the name of the directory to move in the title I think! it's not specified in the feature request but you can move a not selected folder then it's better to show the user which folder will be moved. 
* Use activeDirectory to get the name
* give a `title` prop in MoveDialog which could be a `FormattedMessage` component with values, see intl documentation

Reply:
done